settings.gradle

```
@@ -5,6 +5,45 @@ pluginManagement {
     }
 }
 
+dependencyResolutionManagement {
+    def slangDependenciesVersion = '1.16.0.6211'
+    def analyzerCommonsVersion = '2.13.0.3004'
+    def pluginApiVersion = '10.10.0.2391'
+    def sonarqubeVersion = '10.0.0.68432'
+    def orchestratorVersion = '3.42.0.312'
+    def sonarlintVersion = '9.0.0.74282'
+    // slf4j is provided by SQ, SC or SL, should be aligned with sonar-plugin-api
+    def slf4jApiVersion = '1.7.30'
+
+    versionCatalogs {
+        libs {
+            library("sonar-plugin-api", "org.sonarsource.api.plugin", "sonar-plugin-api").version(pluginApiVersion)
+            library("sonar-xml-parsing", "org.sonarsource.analyzer-commons", "sonar-xml-parsing").version(analyzerCommonsVersion)
+            library("sonar-analyzer-commons", "org.sonarsource.analyzer-commons", "sonar-analyzer-commons").version(analyzerCommonsVersion)
+            library("slang-api", "org.sonarsource.slang", "slang-api").version(slangDependenciesVersion)
+            library("slang-checks", "org.sonarsource.slang", "slang-checks").version(slangDependenciesVersion)
+            library("slang-plugin", "org.sonarsource.slang", "slang-plugin").version(slangDependenciesVersion)
+            library("slf4j-api", "org.slf4j", "slf4j-api").version(slf4jApiVersion)
+            library("scalameta", "org.scalameta", "scalameta_2.13").version("4.9.4")
```

Can we upgrade to [4.9.9](https://mvnrepository.com/artifact/org.scalameta/scalameta_2.13/4.9.9)?
```suggestion
            library("scalameta", "org.scalameta", "scalameta_2.13").version("4.9.9")
```